tests/test_custom_models.py

```diff
@@ -1046,6 +1060,9 @@ def test_disable_adapters_with_merging(self, test_name, model_id, config_cls, co
         if issubclass(config_cls, IA3Config) and model_id == "Conv2d":  # more instability with Conv2d + IA3
             atol, rtol = 1e-3, 1e-3
 
+        if config_kwargs.get("use_dora"):
+            atol, rtol = 1e-3, 1e-3
```

Comment:
Too bad that standard tolerance does not work. However, for me this passed with 1e-4, could you test that? Also, let's only increase the tolerance for the DoRA + embeddings tests.

Reply:
Will test and update.